index.html

```diff
@@ -865,10 +919,86 @@ <h4>issuance</h4>
 <section id="conformance"></section>
 
 <section>
-<h2>Privacy Considerations</h2>
+  <h2>Privacy Considerations</h2>
+  <p>
+    Verifiable Credentials often contain sensitive information that should be protected 
+    to ensure the privacy and security of individuals. This section outlines some privacy 
+    considerations that implementers and users of this specification should be aware of.
+  </p>
+  <p>
+    Implementers SHOULD note and abide by all privacy considerations called out in the [[VC-DATA-MODEL]].
+  </p>
+  <p>
+    In addition to privacy recommendations noted in the [[VC-DATA-MODEL]] the following 
+    considerations should be explicity observed:
+    <ul>
+      <li>
+        <p>
+          Minimization of data: Verifiable Credentials should only contain the minimum amount 
+          of data necessary to achieve their intended purpose. This can help to limit the amount 
+          of sensitive information that is shared or stored unnecessarily.
+        </p>
+      </li>
+      <li>
+        <p>
+          Informed consent: Individuals should be fully informed about how their data will be 
+          used and should have the ability to consent to or decline the use of their data. 
+          This can help to ensure that individuals maintain control over their own personal information.
+        </p>
+      </li>
+      <li>
+        <p>
+          Data protection: Verifiable Credentials should be protected using strong encryption and other 
+          security measures to prevent unauthorized access, modification, or disclosure. 
+        </p>
+      </li>
+    </ul>
+  </p>
+  <p>
+    These considerations are not exhaustive, and implementers and users should consult additional privacy 
+    resources and best practices to ensure the privacy and security of Verifiable Credentials implemented
+    using VC-JWT.
```

Comment:
It would be nice to close with a nice list of NIST references.

Reply:
added #81 to track this